test/exp_test.exs

```diff
@@ -0,0 +1,55 @@
+defmodule ExpTest do
+  use ExUnit.Case
+  doctest MPDecimal
```

Comment:
We don't need each test file to call `doctest MPDecimal`. We only need to execute doctests once for a module. Let's re-create the `test/mpdecimal_test.exs` file and call `doctest MPDecimal` there. That way we'll have one test file for any doctests and each of the functions broken into their own test files?

Reply:
Thanks will do Mike.